scionlab/urls.py

```diff
@@ -79,6 +79,9 @@
     path('api/v2/host/<slug:uid>/deployed_config_version',
          PostHostDeployedConfigVersion.as_view(),
          name='api_post_deployed_version'),
+    path('api/v2/topology/topology.json',
```

Comment:
When I see a file server with an extension, I usually expect that a static file is being served.
Also the file format might change, so the content type in the response should indicate what is being returned (JsonResponse does that), so remove the `.json` from the path.